app/src/main/python/gogdl/dl/managers/manager.py

```
@@ -0,0 +1,207 @@
+                self.logger.info("Using V1Manager for generation 1 game")
+                manager = v1.Manager(self)  # Pass self like V2 does
+            elif generation == 2:
+                self.logger.info("Using V2Manager for generation 2 game")
+                manager = v2.Manager(self)
+            else:
+                raise Exception(f"Unsupported generation: {generation}")
+            
+            manager.download()
+                
+        except Exception as e:
+            self.logger.error(f"Download failed: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)